test/unit/cli.test.ts

```diff
@@ -25,9 +25,9 @@
  --------------
  ******/
 
-import { Server } from '@hapi/hapi'
 import Config from '~/shared/config'
 import { Handlers } from '~/server'
+import { Server } from '@hapi/hapi'
```

Comment:
barring style, is there a specific reason for the shift?

Reply:
I would like to keep in `/server` only the code related to server setup and `handlers` on the same level where `model` and `domain` is placed so we get more flat folder structure

Reply:
@akapur99 barring style fixed